### PR TITLE
Update alerts.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated properties.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5876

File: microsoft-365/security/defender-endpoint/alerts.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ incidentId | Nullable Long | The [Incident](view-incidents-queue.md) ID of the A
 investigationId | Nullable Long | The [Investigation](automated-investigations.md) ID related to the Alert.
 investigationState | Nullable Enum | The current state of the [Investigation](automated-investigations.md). Possible values are: 'Unknown', 'Terminated', 'SuccessfullyRemediated', 'Benign', 'Failed', 'PartiallyRemediated', 'Running', 'PendingApproval', 'PendingResource', 'PartiallyInvestigated', 'TerminatedByUser', 'TerminatedBySystem', 'Queued', 'InnerFailure', 'PreexistingAlert', 'UnsupportedOs', 'UnsupportedAlertType', 'SuppressedAlert'.
 assignedTo | String | Owner of the alert.
+rbacGroupName | String | RBAC device group name
+mitreTechniques | String | Mitre Enterprise technique ID
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/alerts.md]:
```suggestion
rbacGroupName | String | RBAC device group name.
mitreTechniques | String | Mitre Enterprise technique ID.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba Please copy revierw

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**